template.md

```diff
@@ -20,8 +20,10 @@ Deadline for advancement eligibility: [**CALCULATE TIME HERE**](https://www.time
   - <sub>Note: this time is selected to be precisely 10 days prior to the start of the meeting</sub>
 
 1. Proposals not looking to advance may be added at any time; if after the deadline, please always use a pull request so that members are notified of changes.
-1. Proposals looking to advance to stage 0 or 1 must be added (and noted as such) prior to the deadline, or else delegates may object to advancement solely on the basis of missing the deadline.
+1. Proposals seeking feedback at stage 0 must be added (and noted as such) prior to the deadline, or else delegates may object to advancement solely on the basis of missing the deadline.
     1. Such proposals *should* include supporting materials when possible.
+1. Proposals looking to advance to stage 1 must be added (and noted as such) prior to the deadline, or else delegates may object to advancement solely on the basis of missing the deadline.
+    1. Such proposals *must* link to a proposal repository and they *should* link to supporting materials when possible.
```

Comment:
Judging from what follows, the proposal repository _is_ an example of necessary supporting materials—perhaps the stage 1 case should just be merged into the stage 2/3/4 case below?